spec/core_functions/function_exists.hrx

```diff
@@ -149,3 +170,32 @@ Error: wrong number of arguments (2 for 1) for `function-exists'
 >> a {b: function-exists(foo, bar)}
 
    ------^
+
+<===>
+================================================================================
+<===> conflict/options.yml
+---
+:todo:
+- libsass # sass/libsass#2807
+:ignore_for:
+- ruby-sass
+
+<===> conflict/input.scss
+@use "other1" as *;
+@use "other2" as *;
+
+a {b: function-exists(member)}
+
+<===> conflict/other1.scss
+@function member() {@return from other1}
+
+<===> conflict/other2.scss
+@function member() {@return from other2}
+
+<===> conflict/error
+Error: Multiple global modules have a function named "member".
```

Comment:
I don't see where this error is coming from in dart-sass but could it include the names of the modules that have conflicting functions? It would make it easier to track down when many modules are used.

Reply:
There's already a TODO for that: https://github.com/sass/dart-sass/blob/5a64dec4ec54ef4fc8fdf0ca0c16a83fec81463b/lib/src/async_environment.dart#L589